Source/Plugins/CoherenceViewer/CoherenceNode.h

```diff
@@ -113,11 +115,13 @@ class CoherenceNode : public GenericProcessor, public Thread
     Array<int> group1Channels;
     Array<int> group2Channels;
 
+    uint32 validSubProcFullID;
+
```

Comment:
I put this here as a reminder that we should implement something similar to what's in `SampleMathEditor` to enforce that all active channels come from the same processor and subprocessor, and thus have the same sample rate. See `SampleMathEditor::channelChanged`. The difference would be that we want all the active channels to match each other, not a specific channel that's selected on the editor as in Sample Math, so the logic will be a little different. I guess you'd have to deselect all channels from one subprocessor before selecting any from a different subproc.